src/JustEat.StatsD/TagsFormatters/InfluxDbTagsFormatter.cs

```diff
@@ -0,0 +1,29 @@
+namespace JustEat.StatsD.TagsFormatters
+{
+    /// <summary>
+    /// Formats StatsD tags for InfluxDB.
+    /// Tags placed right after the bucket name with format: <code>"," + tag1=value1,tag2,tag3=value</code>.
+    /// </summary>
+    public sealed class InfluxDbTagsFormatter : StatsDTagsFormatter
```

Comment:
As we have the static properties in the public class, might be better to make these concrete implementations `internal` so we don't have to worry about breaking changes to the public surface area in the future if we wanted to do some refactoring for all these different providers.

Reply:
Done!

Reply:
There is a small allocation when using the tags.

With `Dictionary`:

``` ini

BenchmarkDotNet=v0.12.1, OS=Windows 10.0.18363.1082 (1909/November2018Update/19H2)
Intel Core i7-7700HQ CPU 2.80GHz (Kaby Lake), 1 CPU, 8 logical and 4 physical cores
.NET Core SDK=5.0.102
  [Host]     : .NET Core 3.1.9 (CoreCLR 4.700.20.47201, CoreFX 4.700.20.47203), X64 RyuJIT
  DefaultJob : .NET Core 3.1.9 (CoreCLR 4.700.20.47201, CoreFX 4.700.20.47203), X64 RyuJIT


```
|                                   Method |     Mean |   Error |  StdDev | Gen 0 | Gen 1 | Gen 2 | Allocated |
|----------------------------------------- |---------:|--------:|--------:|------:|------:|------:|----------:|
|                              BufferBased | 239.9 ns | 1.71 ns | 1.43 ns |     - |     - |     - |         - |
|                  BufferBasedIgnoringTags | 272.6 ns | 1.75 ns | 1.64 ns |     - |     - |     - |         - |
|             BufferBasedIgnoringEmptyTags | 250.3 ns | 1.95 ns | 1.82 ns |     - |     - |     - |         - |
|  BufferBasedWithTagsFormatterAndNullTags | 241.7 ns | 1.69 ns | 1.58 ns |     - |     - |     - |         - |
|             BufferBasedWithTagsFormatter | 667.8 ns | 2.55 ns | 2.13 ns |     - |     - |     - |         - |
| BufferBasedWithTagsFormatterAndEmptyTags | 253.1 ns | 2.59 ns | 2.43 ns |     - |     - |     - |         - |

With `IDictionary`:

``` ini

BenchmarkDotNet=v0.12.1, OS=Windows 10.0.18363.1082 (1909/November2018Update/19H2)
Intel Core i7-7700HQ CPU 2.80GHz (Kaby Lake), 1 CPU, 8 logical and 4 physical cores
.NET Core SDK=5.0.102
  [Host]     : .NET Core 3.1.9 (CoreCLR 4.700.20.47201, CoreFX 4.700.20.47203), X64 RyuJIT
  DefaultJob : .NET Core 3.1.9 (CoreCLR 4.700.20.47201, CoreFX 4.700.20.47203), X64 RyuJIT


```
|                                   Method |     Mean |    Error |   StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|----------------------------------------- |---------:|---------:|---------:|-------:|------:|------:|----------:|
|                              BufferBased | 255.5 ns |  5.09 ns |  5.44 ns |      - |     - |     - |         - |
|                  BufferBasedIgnoringTags | 289.0 ns |  5.68 ns |  9.33 ns |      - |     - |     - |         - |
|             BufferBasedIgnoringEmptyTags | 257.9 ns |  3.33 ns |  2.60 ns |      - |     - |     - |         - |
|  BufferBasedWithTagsFormatterAndNullTags | 257.4 ns |  4.81 ns |  8.30 ns |      - |     - |     - |         - |
|             BufferBasedWithTagsFormatter | 912.2 ns | 16.23 ns | 18.04 ns | 0.0534 |     - |     - |     168 B |
| BufferBasedWithTagsFormatterAndEmptyTags | 274.3 ns |  5.46 ns |  9.42 ns |      - |     - |     - |         - |